app/collections/show-list.cjsx

```diff
@@ -21,10 +21,13 @@ module?.exports = React.createClass
   displayName: 'CollectionShowList'
   mixins: [History]
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
   componentDidMount: ->
     @fetchCollectionSubjects pick @props.location.query, VALID_COLLECTION_MEMBER_SUBJECTS_PARAMS
 
-  componentWillReceiveProps: (nextProps) ->
+  componentWillReceiveProps: (nextProps, nextContext) ->
```

Comment:
When `contextTypes` are defined, they also become available in the lifecycle methods. While not explicitly used, it makes it clear it's now available in the lifecycle: https://facebook.github.io/react/docs/context.html#referencing-context-in-lifecycle-methods